purchases/src/main/kotlin/com/revenuecat/purchases/amazon/handler/PurchaseHandler.kt

```diff
@@ -38,26 +38,25 @@ internal class PurchaseHandler(
         activity: Activity,
         appUserID: String,
         storeProduct: StoreProduct,
-        presentedOfferingIdentifier: String?,
+        presentedOfferingContext: PresentedOfferingContext?,
```

Comment:
Is this not being used in this method? 🤔

Reply:
Nope, it was actually unused. I removed part of it but forgot to remove the parameter here. Thanks!